.env

```diff
@@ -2,4 +2,7 @@ DB_USERNAME = sen-mairie
 DB_PASS=1234
 NODE_DEV=dev
 PORT=8002
-DB_URL=mongodb+srv://someone:someone@e-commune.hlth8.mongodb.net/ecommune?retryWrites=true&w=majority
```

Comment:
we have to remove the .env file from being pushed.
I created a .env.local file where you can find the same keys as in the .env file. So be upset that when you clone this project please copy the .env file content and past it in you created .env file and change the values.